tests/php/ObjectCacheTest.php

```
@@ -327,4 +327,101 @@ public function test_array_salt_order_changes_normalized_salt(): void {
+				// phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_mkdir -- Test fixture directory.
+				mkdir( $helper_dir, 0777, true );
+			}
+			// phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_file_put_contents -- Test fixture file.
+			file_put_contents(
+				$helper_dest,
+				"<?php\nif ( ! function_exists( 'wppo_redis_connect' ) ) {\n\tfunction wppo_redis_connect( \$config ) {\n\t\treturn new \\stdClass();\n\t}\n}\nif ( ! function_exists( 'wppo_apply_redis_options' ) ) {\n\tfunction wppo_apply_redis_options( \$redis, \$config ) {}\n}\n"
+			);
+
+			\Brain\Monkey\Functions\when( 'is_wp_error' )->justReturn( false );
+
+			$instance2 = ( new \ReflectionClass( 'WP_Object_Cache' ) )->newInstanceWithoutConstructor();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '407', column '23'). (undefined)

(MissingImport)